sdk/digitaltwins/test-resources.bicep

```diff
@@ -87,3 +92,76 @@ resource digitaltwinEndpoints 'Microsoft.DigitalTwins/digitalTwinsInstances/endp
 }
 
 output DIGITALTWINS_URL string = 'https://${digitaltwin.properties.hostName}'
+
+
+var containerName = baseName
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2022-05-01' = {
```

Comment:
this section sets up the new test resources required by Import API. The input blob name and storage container endpoint are used in TestEnvironment.cs.